### PR TITLE
Add `Tidefi`

### PR DESCRIPTION
**Pull request by lemarier**
Related to;

https://github.com/tidelabs/tidechain
https://github.com/tidelabs/tidext

PS. Website will be launched soon with the official public announcement.

Thanks guys!

### CODE REVIEW

File: ss58-registry.json [hunk 1]
```diff
@@ -802,6 +802,24 @@
       "standardAccount": "*25519",
       "website": "https://subspace.network"
     },
+    {
+      "prefix": 7006,
+      "network": "tidefi_testnet",
```
Comment (bkchr) [inline ss58-registry.json:807]: Test nets don't need their own prefix.

Comment (lemarier) [reply ss58-registry.json]: Perfect! I've removed it -- Thanks :)

**Comment by nothingismagick** [comment]
@bkchr - is this appropriate now? thanks :)

**Comment by bkchr** [comment]
Yeah ty, now it is up to @BenWhiteJam

**Review by BenWhiteJam — APPROVED** [review]

**Comment by bkchr** [comment]
@lemarier please merge master.

File: Cargo.toml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 [package]
 name = "ss58-registry"
 authors = ["Parity Technologies <admin@parity.io>"]
-version = "1.17.0"
+version = "1.18.0"
```
Comment (bkchr) [inline Cargo.toml:4]:
```suggestion
version = "1.20.0"
```

**Review by bkchr — APPROVED** [review]

**Pull request merged**